src/common/hash/index.ts

```diff
@@ -0,0 +1,8 @@
+export function truncHashPortion(str, firstCharCount = str.length, endCharCount = 0, dotCount = 3) {
+	var convertedStr="";
+	convertedStr+=str.substring(0, firstCharCount);
+	convertedStr += ".".repeat(dotCount);
```

Comment:
Suggestion: use the `…` character instead (https://en.wikipedia.org/wiki/Ellipsis). This is a single charater that is smaller than three separate dots. Then you don't need the `dotCount`.

Reply:
@webmaster128 thanks for the response, did it